burn-tensor/src/tests/module/maxpool1d.rs

```diff
@@ -9,7 +9,7 @@ mod tests {
     #[test]
```

Comment:
Is there a reason to update the tests for max pool 1d? Maybe updating one test with no padding is good.

Reply:
The idea is to have at least one test for the most basic scenario. With what candle offers as of now it's not possible to put padding in max pooling, so if every test has padding I cannot check that I've done the basic binding correctly.